src/current/v24.3/wal-failover.md

```diff
@@ -0,0 +1,495 @@
+
+Next, make sure the value of the [`storage.wal_failover.unhealthy_op_threshold`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-wal-failover-unhealthy-op-threshold) cluster setting is a duration less than the [`storage.max_sync_duration`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-max-sync-duration) cluster setting:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING storage.wal_failover.unhealthy_op_threshold = '${value less than storage.max_sync_duration}'
+~~~
+
+The recommended value of the [`storage.wal_failover.unhealthy_op_threshold`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-wal-failover-unhealthy-op-threshold) cluster setting is `100ms`.
+
+### 8. Can I enable WAL failover during runtime for a single data store cluster?
+
+Yes: If the side disk for WAL failover was configured before the cluster was started, set the [`storage.wal_failover.unhealthy_op_threshold`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-wal-failover-unhealthy-op-threshold) cluster setting to a value less than the [`storage.max_sync_duration`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-max-sync-duration) cluster setting to enable WAL failover.
+
+No: If the side disk for WAL failover was **not** pre-configured before the cluster was started, changing the value of the [`storage.wal_failover.unhealthy_op_threshold`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-wal-failover-unhealthy-op-threshold) cluster setting to something less than the [`storage.max_sync_duration`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-max-sync-duration) cluster setting **will not have any effect and when the disk stalls, WAL failover will not trigger**. In other words, you cannot enable WAL failover at runtime for a single store cluster if you did not [pre-configure a side disk](#single-store-config).
+
+The recommended default value of the [`storage.wal_failover.unhealthy_op_threshold`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-wal-failover-unhealthy-op-threshold) cluster setting is `100ms`.
+
+### 9. Can I change the log buffering on a node before enabling WAL failover?
+
+Yes.
+
+### 10. Does configuring WAL Failover on a running cluster require rolling restarts?
+
+Yes: For a single-store cluster, if you did not have a side disk pre-configured for WAL failover, you will need to do a rolling restart.
+
+Yes: For a multi-store cluster, if you are using file-based logging and did not have log buffering configured according to [3. Log Configuration for WAL Failover](#3-log-configuration-for-wal-failover), you will need to do a rolling restart.
+
+No: In all other cases, you can enable WAL failover without doing a rolling restart.
```

Comment:
I don't understand this "frequently-asked question". You must restart the process to begin to pass `--wal-failover=among-stores` or `COCKROACH_WAL_FAILOVER=among-stores`. IMO, that's what it means to "enable" it.

Reply:
removed it! (and renumbered around it)